internal/k8s/ako_init.go

```diff
@@ -1050,7 +1030,26 @@ func (c *AviController) FullSyncK8s(sync bool) error {
 				}
 			}
 		}
-
+		//Ingress Section
+		if utils.GetInformers().IngressInformer != nil {
+			for namespace := range acceptedNamespaces {
+				ingObjs, err := utils.GetInformers().IngressInformer.Lister().Ingresses(namespace).List(labels.Set(nil).AsSelector())
+				if err != nil {
+					utils.AviLog.Errorf("Unable to retrieve the ingresses during full sync: %s", err)
+				} else {
+					for _, ingObj := range ingObjs {
+						key := utils.Ingress + "/" + utils.ObjKey(ingObj)
+						meta, err := meta.Accessor(ingObj)
+						if err == nil {
+							resVer := meta.GetResourceVersion()
+							objects.SharedResourceVerInstanceLister().Save(key, resVer)
+						}
+						utils.AviLog.Debugf("Dequeue for ingress key: %v", key)
+						nodes.DequeueIngestion(key, true)
+					}
+				}
+			}
+		}
```

Comment:
Shall we add it at the end after if - else condition since this is a duplicate code?

Reply:
For WCP, `Gateway` being parent for `Ingress`, we might need to check impact on performance of adding that at the end. To keep that order of object creation same, I added that duplicate code.

Reply:
sounds good.

@abhishekbsingh, could you please review this PR on priority?